package.json

```diff
@@ -18,7 +18,10 @@
     "build-server": "tsc --project tsconfig.server.json"
   },
   "dependencies": {
+    "@coderline/alphatab": "github:coderline/alphatab",
+    "@types/mongodb": "^3.6.10",
```

Comment:
These should be in the devDependencies, and I'm not really sure if you need the `@coderline/alphatab` dependency at all

Reply:
Ich hab die dependency deinstalliert. Ich denke das ich die später aber noch benötige.
`npm uninstall  @coderline/alphatab`